recipes-devtools/rust/rust_1.12.1.bb

```diff
@@ -0,0 +1,26 @@
+inherit rust-installer
+require rust.inc
+require rust-source-${PV}.inc
+
+DEPENDS += "cmake-native"
```

Comment:
I can't find the issue for it but I've raised it with upstream and the patch is stuck in rollup queues. It didn't seem harmful to include it since rust-llvm depends on it so we've already built it.